### ISSUE TITLE
[a11y]: Activating a switch or checkbox with TalkBack no longer gives feedback about state

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#17775]
Turn TalkBack on and double tap on a checkbox or switch. It is supposed to announce how the state changed. However, no announcement is made anymore.

/cc @jonahwilliams

**Comment by goderbauer** [comment]
Maybe caused by https://github.com/flutter/engine/pull/5081/files, which changed the event handling?

**Comment by goderbauer** [comment]
Affirmative, if  https://github.com/flutter/engine/pull/5081 is reverted, this starts working again :/

**Comment by jonahwilliams** [comment]
Was this fixed by https://github.com/flutter/engine/pull/5330 ?

**Comment by goderbauer** [comment]
No, unfortunately not. flutter/engine#5330 was fixing long-clicks for non-TalkBack a11y services.

**Comment by jonahwilliams** [comment]
Okay, I can prioritize this for Thursday evening/Friday once I have access to a device for debugging.  

@goderbauer does this happen only when talkback is turned on when the device is running?  If so it might be an easy fix related to how we detect whether semantics are enabled.  Otherwise I might need to do a partial rollback of the tap behavior.

**Comment by goderbauer** [comment]
Not 100% sure I understand the question. I have TalkBack enabled on my phone before launching the gallery. TalkBack works as expected. However, when I a11y focus on a Checkbox and then double tap on the screen it does not announce that the state of the checkbox has changed.

This is on a Nexus 6P with Android 7.1.2 and the latest TalkBack.

**Comment by jonahwilliams** [comment]
I completely forgot that this change was reverted in https://github.com/flutter/flutter/pull/17269.  @cbracken do you have more context here or should I try again?

**Comment by jonahwilliams** [comment]
I'm not able to reproduce any failures on the gallery semantics tests locally - I believe those were the ones that were failing.  I think the failure might have been fixed in the gallery updates, but there is a possibility that it only shows up on certain hardware.

In this case, I could continue testing locally or we could push and see if there are failures.  I don't like this option but running these devicelab and other tests locally is very manual (unless there is a magic test script I am missing) and I am bound to miss something

**Comment by cbracken** [comment]
> unless there is a magic test script I am missing

Right now, the only way to run devicelab tests locally is:
```
cd dev/devicelab
pub get
dart bin/run.dart -t TASK_NAME
```

If you've done that locally for the tests that were problematic, I'd re-land it (once the build is green again).
